mapadroid/madmin/endpoints/api/apks/MadApkEndpoint.py

```diff
@@ -39,6 +38,10 @@ async def get(self):
             except KeyError:
                 return await self._json_response(data=data[apk_type])
 
+    def allowed_file(self, filename):
+        ALLOWED_EXTENSIONS = set(['apk', 'apkm', 'zip'])
```

Comment:
This line is equivalent to `ALLOWED_EXTENSIONS = {'apk', 'apkm', 'zip'}` which is more efficient as it doesnt create a list then convert it to a set. I dont think it matters in this context though.

This should be a global variable thats initialized once and referenced instead of initialized on every call.